src/azul/plugins/metadata/hca/transform.py

```diff
@@ -80,6 +83,43 @@
 pass_thru_uuid4: PassThrough[api.UUID4] = PassThrough()
 
 
+class OrganismAge(FieldType[JSON, str]):
+
+    def to_index(self, value: JSON) -> str:
+        try:
+            value, unit = value['value'], value['unit']
+        except TypeError:  # Used for translating None
+            value, unit = [None, None]
+        except KeyError:
+            require({'value', 'unit'} == value.keys(),
+                    'A filter for `organismAge` must specify a dictionary with '
```

Comment:
This can't mention filters because it is not used exclusively on filters. You need to word this generically so it works in all use cases.